README.md

```diff
@@ -38,34 +38,84 @@ The key/value pairs insides this ConfigMap are:
+              x-descriptors:
+                - servicebinding 
+          ```
+
+2. Non-OLM Operator: - An annotation in the Operator's CRD to mark which `status` and/or `spec` properties reference the [binding data](#service-binding-schema).  The value of this annotation can be specified in either [JSONPath](https://kubernetes.io/docs/reference/kubectl/jsonpath/) or [GO templates](https://golang.org/pkg/text/template/):
       * ConfigMap:
-        * servicebinding/configMap: {.status.bindable.ConfigMap}
+        *   ```
+             "servicebinding.dev/certificate":
+             "path={.status.data.dbConfiguration},objectType=ConfigMap"
+            ```
+
       * Secret:
-        * servicebinding/secret: {.status.bindable.Secret}
-      * Individual binding items:
-        * servicebinding/secret/host: {.status.address}
-        * servicebinding/secret/`<binding_property>`: {.status.`<status_property>}` (where `<binding_property>` is any property from the binding schema, and `<status_property>` refers to the path to the correspoding `status` property)
-
-3. Regular k8s Deployment (Ingress, Route, Service, Secret, ConfigMap etc)  - An annotation in the corresponding CR that maps the `status`, `spec` or `data` properties to their corresponding [binding data](#service-binding-schema). The value of this annotation can be specified in either [JSONPath](https://kubernetes.io/docs/reference/kubectl/jsonpath/) or [GO templates](https://golang.org/pkg/text/template/):
-      * servicebinding/secret/host: {.status.ingress.host}
-      * servicebinding/secret/host: {.status.address}
-      * servicebinding/secret/`<binding_property>`: `<property_path>` (where `<binding_property>` is any property from the binding schema, and `<property_path>` refers to the path to the correspoding `status`, `spec` or `data` property)
+        *   ```
+             "servicebinding.dev/dbCredentials":
+             "path={.status.data.dbCredentials},objectType=Secret"
+            ```
+
+      * Individual binding items from a `ConfigMap`
+        *   ```
+            “servicebinding.dev/host": 
+            “path={.status.data.dbConfiguration},objectType=ConfigMap,sourceKey=address"
+            ```
+     
+        *   ```
+            “servicebinding.dev/port": 
+            “path={.status.data.dbConfiguration},objectType=ConfigMap
+            ```
+
+      * Individual backing items from a path referencing a string value
+
+        * ```
+            “servicebinding.dev/uri”:"path={.status.data.connectionURL}"
+          ```
+
+      
+3. Regular k8s resources (Ingress, Route, Service, Secret, ConfigMap etc)  - An annotation in the corresponding Kubernetes resources that maps the `status`, `spec` or `data` properties to their corresponding [binding data](#service-binding-schema). The value of this annotation can be specified in either [JSONPath](https://kubernetes.io/docs/reference/kubectl/jsonpath/) or [GO templates](https://golang.org/pkg/text/template/)
```

Comment:
Same comment regarding `The value of this annotation can be specified in either [JSONPath](https://kubernetes.io/docs/reference/kubectl/jsonpath/) or [GO templates](https://golang.org/pkg/text/template/)`

Reply:
Thanks, done.